test/v2/removeLiquidity/composableStable.recovery.integration.test.ts

```diff
@@ -105,29 +105,22 @@ export class MockApi {
     public async getPool(id: Hex): Promise<PoolState> {
         const tokens = [
             {
-                address:
-                    '0x156c02f3f7fef64a3a9d80ccf7085f23cce91d76' as Address, // swETH-bb-a-WETH-BPT
-                decimals: 18,
+                ...TOKENS_MAINNET.swETH_bb_a_WETH_BPT,
```

Comment:
I considered using the spread operator here, but I suggest this small change so we're more explicit from what we expect from the API.
```suggestion
                address: TOKENS_MAINNET.swETH_bb_a_WETH_BPT.address,
                decimals: TOKENS_MAINNET.swETH_bb_a_WETH_BPT.decimals,
```